packages/veritone-widgets/src/shared/VeritoneApp.js

```diff
@@ -38,6 +38,10 @@ class _VeritoneApp {
     if (OAuthToken) {
       this._store.dispatch(authModule.setOAuthToken(OAuthToken));
     }
+
+    if (!OAuthToken && !sessionToken) {
+      this._store.dispatch(authModule.checkAuthNoToken());
```

Comment:
thinking about it, we should probaby make this whole thing an if-else-if-else chain, since you wouldn't reasonably provide both tokens

if (sessionToken)
else if (oauthToken)
else { checkAuthNoToken }